rules/override/shellcode.yara

```diff
@@ -0,0 +1,122 @@
+rule shellcode_hash : override {
+    meta:
+        malware_shellcode_hash = "high"
```

Comment:
I couldn't find a rule with this name:

https://github.com/search?q=repo%3Achainguard-dev%2Fmalcontent+malware_shellcode_hash&type=code

Reply:
It exists in two places:
- https://github.com/chainguard-dev/malcontent/blob/0acb2e0e98915c8fee1d6723dfbf1a29d861ed5c/third_party/yara/JPCERT/shellcode.yara#L1
- https://github.com/chainguard-dev/malcontent/blob/0acb2e0e98915c8fee1d6723dfbf1a29d861ed5c/third_party/yara/JPCERT/other/shellcode.yara#L1

The override works across both, though:
```
HIGH  3P/JPCERT/shellcode/hash       detect shellcode api hash value, by JPCERT/CC Incident  $or60hAddShl1Hash32_GetProcAddress
                                     Response Group                                          $or60hAddShl1Hash32_LoadLibraryA
```